server/neptune/workflows/internal/deploy/terraform/state.go

```diff
@@ -48,18 +49,18 @@ func (n *StateReceiver) Receive(ctx workflow.Context, c workflow.ReceiveChannel,
 		}
 	}
 
+	// Updates all other deployments waiting in queue when the current deployment is pending a confirm/reject user action. Current deployment is not on the queue at this point since its child TerraformWorkflow was started.
 	if workflowState.Apply != nil &&
 		workflowState.Apply.Status == state.WaitingJobStatus &&
-		!reflect.ValueOf(workflowState.Apply.OnWaitingActions).IsZero() {
-		// update queue with information about current deployment pending confirm/reject action
-		infos := n.Queue.GetOrderedMergedItems()
+		len(workflowState.Apply.OnWaitingActions.Actions) > 0 {
+		queued_deployments := n.Queue.GetOrderedMergedItems()
```

Comment:
```suggestion
		queuedDeployments := n.Queue.GetOrderedMergedItems()
```
nit: camel casing is convention in go 

my spicy take though: I do tend to like snake case better, I think its more legible, and sometimes I use it in tests. But it's convention to use camel casing general for go